account_credit_control_payment_details/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================================
+Account Credit Control Payment Details
+======================================
+
+For credit control payment details in PDF report, Go to ``Accounting > Periodical Processing > Credit
```

Comment:
Description module and its usage are mixed up! Coul you please arrange them?

Thank you in advance.